cmake/AddFdbTest.cmake

```diff
@@ -375,3 +375,92 @@ function(package_bindingtester)
   add_custom_target(bindingtester ALL DEPENDS ${tar_file})
   add_dependencies(bindingtester copy_bindingtester_binaries)
 endfunction()
+
+function(add_fdbclient_test)
```

Comment:
Assuming most of the audience would be terrible at reading CMake files like me, can we add comment here (and other 2 macros), on what each of them does. If I'm not mistaken:
- add_fdbclient_test: Single cluster integration test.
- add_fdb_multiclient_test: Multicluster integration test
- add_java_test: Java unit tests that doesn't require a FDB cluster running.

On side note, why can't `add_fdb_client_test` be a special case of `add_muticlient_test` where number of clusters = 1.

Reply:
It could be, but I didn't write `add_fdbclient_test`, and I honestly don't understand it very well, so I just left it as is.